FlowCrypt/src/androidTest/assets/messages/info/standard_msg_info_plain_text_single_to_replyto_cc.json

```diff
@@ -0,0 +1,53 @@
+{
+  "encryptionType": "STANDARD",
+  "inlineSubject": "Standard message - plane text",
```

Comment:
Tiny nitpick - I believe 'plane text' should be 'plain text', if you mean human readable characters, or 'plaintext' if you mean unencrypted text.

If it's intentional then I'm sorry for my misunderstanding, otherwise if it's a misspelling then I can add a commit changing it.

Reply:
taking into account file name that is really likely should have been "plain text" or even "plaintext" (as a single word).

Reply:
@seisvelas 
>'plaintext' if you mean unencrypted text.

That's what I meant

>I can add a commit changing it.

Please don't worry. I will do it myself. Thank you for noticing